db/schema.rb

```diff
@@ -224,9 +225,9 @@
     t.uuid "publisher_user_id"
     t.datetime "expiry_time"
     t.string "supporting_documents"
-    t.string "job_roles", array: true
```

Comment:
These sort of things keep springing up in schema changes

Reply:
It's one of the long-lived problems with Rails schemas unfortunately. `job_roles` needs to stay in, doesn't it?

Reply:
Yes definitely, just the order of fields in the schema always seems to jump around when we do migrations